### ISSUE TITLE
turn docs into a separate page

### ISSUE THREAD

**Issue by dlaehnemann** [vembrane/vembrane#174]
Currently, the docs are all in README.md files, and those have gotten rather long. It probably makes sense to turn this into readthedocs or GitHub pages, and give it a clearer sub-structure, where it's also quicker to find stuff.

**Comment by tedil** [comment]
Agreed (as long as the initial README still contains some basic information / usage examples for getting started quickly)
